utils/services/network/FirebaseApi.ts

```diff
@@ -161,6 +167,85 @@ async function addArticle(article: ArticleModel) {
   await setDoc(newDocRef, article);
 }
 
+// Adding article, now using direct firebaseClient
+interface AddArticleProps {
+  data: WritingPanelFormProps;
+  callback?: (
+    resp: MainNetworkResponse<ArticleModel | null | FirebaseError>,
+  ) => void;
+}
+
+async function addArticle1({
```

Comment:
Might need to change the name in the future. Not very descriptive!

Reply:
Yea... addArticle1. Probably should name with top to bottom functionality pattern :


So, instead of `addArticle` which :
- `add` is the main function of the method
- `Article` is the type/topic of the method


We can use 
```suggestion
async function articleAdd({...
```


The same with locating something in disk. You go from the things that are closer to the root then scales down to the bottom, e.g

/article/add

Also this will help when creating modular pattern, which by the way i had already implemented on this one [https://github.com/arunstop/scuffed-blogger/pull/7/commits/52d0d296e6f6d847cb114aa61ead0bd7d849ec24#diff-d57cc378147023378a9f5a548208b959d4e7cbadaa76b5bd9ed2f37ff089b2bd](utils/services/network/FirestoreApi.ts:52)

I think we might have to do a big refactoring. :thinking:

Reply:
Agreed. But not right now though. I think i can do it.